src/main/scala/org/zalando/kanadi/models/PublishedBy.scala

```diff
@@ -0,0 +1,13 @@
+package org.zalando.kanadi.models
+
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder}
+
+final case class PublishedBy(name: String) extends AnyVal
```

Comment:
The API definition has "NOTE: Enriched by Nakadi, should not be specified when publishing event."
This sounds like an implementation detail that might change. I would propose that you make this an optional field

Reply:
The field `published_by` is already `Option[PublishedBy]` in Metadata, should we add another Option here?

Reply:
IMHO it is enough to have it optional in the `Metadata` case class only, given that `PublishedBy` is just a value holder.

Reply:
according to the ticket ARUHA-2896 description, it was done for consumers to see who published the event.

@Yaneeve 
do you have any concerns regarding the current implementation in this PR?

Reply:
My only comment here is to leave it optional (I have had issues in the past where mandatory fields weren't actually mandatory even if documented otherwise in swagger) and make sure the data is correct, i.e. is `pubished_by` actually a string.

Ideally you would write a test where you would get a `Metadata` with a defined `published_by` from the docker nakadi to verify the data structure is correct but this isn't always possible.

Reply:
> My only comment here is to leave it optional

@mdedetrich 
hm.. a bit confused. Are you talking about making it optional 
here:
```
final case class PublishedBy(name: String)
```
?
--
regarding testing with a docker image, I'm afraid it is too old

Reply:
@mdedetrich @Yaneeve 
are there any concerns here that needs to be resolved?

if not, propose to merge and release it

Reply:
@gchudnov I wasn't talking about the value class but the field inside of `Metadata`

Reply:
@mdedetrich 

got it, thank you for the clarification.
yes, the field inside `Metadata` is optional.

any objections to merge-publish?

Reply:
All good from me

Reply:
Thank you!